src/main/webapp/app/exercises/file-upload/participate/file-upload-result.component.ts

```diff
@@ -18,7 +18,7 @@ export class FileUploadResultComponent {
         }
         const [feedbackWithCredits, feedbackWithoutCredits] = partition(result.feedbacks, (feedback) => feedback.credits !== 0);
         this.feedbacks = feedbackWithCredits;
-        this.generalFeedback = feedbackWithoutCredits[0] ? feedbackWithoutCredits[0] : null;
+        this.generalFeedback = feedbackWithoutCredits[0] ? feedbackWithoutCredits[0] : undefined;
```

Comment:
This can be simplified.
```suggestion
        this.generalFeedback = feedbackWithoutCredits[0] || undefined;
```